src/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+function Footer(props) {
+  return (
+    <div className="footer">
+      <ul className="list-group">
+        {props.address.map(element => (
+          <li>{element}</li>
+        ))}
```

Comment:
There's something missing here, can you spot what it is? I think if you look in the JS console in your browser there should be an error message which might help you out 🙂

Reply:
I added key to fix this error.